activiti-engine/src/test/java/org/activiti/engine/test/impl/util/BitMaskUtilTest.java

```diff
@@ -0,0 +1,143 @@
+   * This test case covers:
+   *  - conditional line 82 branch to line 82
+   *  - conditional line 82 branch to line 86
+   *  - conditional line 86 branch to line 86
+   */
+
+  @Test
+  public void isBitOnInputPositivePositiveOutputTrue() {
+
+    // Arrange
+    final int value = 32;
+    final int bitNumber = 6;
+
+    // Act
+    final boolean retval = BitMaskUtil.isBitOn(value, bitNumber);
+
+    // Assert result
+    Assert.assertEquals(true, retval);
+  }
+
+    /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 82 branch to line 82
+   *  - conditional line 82 branch to line 86
+   *  - conditional line 86 branch to line 86
+   */
+
+  @Test
+  public void isBitOnInputZeroPositiveOutputFalse() {
+
+    // Arrange
+    final int value = 0;
+    final int bitNumber = 6;
+
+    // Act
+    final boolean retval = BitMaskUtil.isBitOn(value, bitNumber);
+
+    // Assert result
+    Assert.assertEquals(false, retval);
+  }
+
+    /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 100 branch to line 103
+   *  - conditional line 103 branch to line 103
+   */
+
+  @Test
+  public void setBitInputZeroPositiveFalseOutputZero() {
+
+    // Arrange
+    final int value = 0;
+    final int bitNumber = 8;
+    final boolean bitValue = false;
+
+    // Act
+    final int retval = BitMaskUtil.setBit(value, bitNumber, bitValue);
+
+    // Assert result
+    Assert.assertEquals(0, retval);
+  }
+
+    /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 100 branch to line 101
+   *  - conditional line 101 branch to line 101
+   */
+
+  @Test
+  public void setBitInputZeroPositiveTrueOutputPositive() {
+
+    // Arrange
+    final int value = 0;
+    final int bitNumber = 8;
+    final boolean bitValue = true;
+
+    // Act
+    final int retval = BitMaskUtil.setBit(value, bitNumber, bitValue);
+
+    // Assert result
+    Assert.assertEquals(128, retval);
+  }
+
+    /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 65 branch to line 65
+   *  - conditional line 65 branch to line 70
+   */
+
+  @Test
+  public void setBitOffInputZeroPositiveOutputZero() {
+
+    // Arrange
+    final int value = 0;
+    final int bitNumber = 6;
+
+    // Act
+    final int retval = BitMaskUtil.setBitOff(value, bitNumber);
+
+    // Assert result
+    Assert.assertEquals(0, retval);
+  }
+
+    /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 48 branch to line 48
+   *  - conditional line 48 branch to line 53
+   */
+
+  @Test
+  public void setBitOnInputPositivePositiveOutputPositive() {
+
+    // Arrange
+    final int value = 128;
+    final int bitNumber = 6;
+
+    // Act
+    final int retval = BitMaskUtil.setBitOn(value, bitNumber);
+
+    // Assert result
+    Assert.assertEquals(160, retval);
+  }
```

Comment:
All good 👍 - do we have the case where it throws the exception or is that already covered?